zerver/views/realm.py

```diff
@@ -345,7 +345,7 @@ def update_realm(
             raise JsonableError(str(err.message))
 
         do_change_realm_subdomain(realm, string_id, acting_user=user_profile)
-        data["realm_uri"] = realm.url
+        data["realm_url"] = realm.url
```

Comment:
This looks like an API change. So it should also go in its own commit, with a feature-level bump. Should also get mentioned in `#api design` before we commit to it.

Reply:
Similar as the last comment above, I have merge this [API endpoint change](
https://github.com/zulip/zulip/blob/407b4b4172580c8ded9d181514d888089f43eb9d/zerver/views/realm.py#L345) (`PATCH /realm`) into an existing commit for api related changed. Also documented the change in [changelog](https://github.com/zulip/zulip/blob/407b4b4172580c8ded9d181514d888089f43eb9d/api_docs/changelog.md?plain=1#L27).